docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+      },
+      {
+        name: 'infoText',
+        type: 'string',
+        description: 'Contextual information displayed in a tooltip to describe the Datapoint',
+      },
+      {
+        name: 'size',
+        type: `'md' | 'lg'`,
+        defaultValue: 'md',
+        description: 'Used to set the size of the datapoint',
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection
+    name="Localization"
+    description={`Be sure to localize the \`title\`, \`infoText\` and \`trendAccessibilityLabel\` props. Note that localization can lengthen text by 20 to 30 percent.`}
+  />,
+);
+
+card(
+  <MainSection name="Variants">
+    <MainSection.Subsection columns={2} title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="size=md"
+        defaultCode={`
+        <Datapoint size="md" title="Spend" value="$1.23M" />
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        title="size=lg"
+        defaultCode={`
+        <Datapoint size="lg" title="Spend" value="$1.23M" />
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`Use \`trend\` to display the delta in the value of a Datapoint over time. Make sure to provide an \`accessibilityLabel\` when the trend is above or below zero.`}
+      title="Trend"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Flex direction="column" gap={4}>
+  <Datapoint title="Pin clicks" value="1.23k" trend={{value: 12, accessibilityLabel: "Trending up"}} />
+  <Datapoint title="Saves" value="123" trend={{value: 0, accessibilityLabel: ""}} />
+  <Datapoint title="Total impressions" value="1.23M" trend={{value: -5, accessibilityLabel: "Trending down"}}  />
+</Flex>
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`Use \`trendSignal\` to explicitly the whether the \`trend\` is considered "good", "bad" or "neutral". By default, a positive trend will be considered "good", a negative trend "bad" and a trend of 0 "neutral".`}
+      title="Trend signal"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Flex direction="column" gap={4}>
+  <Datapoint title="Bounce rate" value="86.3%" trend={{value: 29, accessibilityLabel: "Trending up"}} trendSignal="bad" />
+  <Datapoint title="Conversion falloffs" value="92" trend={{value: -10, accessibilityLabel: "Tending down"}} trendSignal="good" />
+  <Datapoint title="Spend" value="$19.3k" trend={{value: -4, accessibilityLabel: "Trending down"}} trendSignal="neutral"  />
+</Flex>
+`}
+      />
+    </MainSection.Subsection>
+
+    <MainSection.Subsection
+      description={`The \`infoText\` prop is intended to provide the user context, detail and/or framing for a Datapoint.`}
```

Comment:
"...framing for a Datapoint through a [Tooltip](/Tooltip)."

Reply:
Done!

Reply:
Done!